src/workspaceMonitor.ts

```diff
@@ -172,6 +172,15 @@ export class WorkspaceMonitor implements vscode.Disposable {
 	private maybeNotifyOutdated(workspace: Workspace) {
 		if (!this.notifiedOutdated && workspace.outdated) {
 			this.notifiedOutdated = true;
```

Comment:
Not a huge deal but can we move this after the `disableNotifications` check?  I think we should only set `notifiedOutdated` if we did in fact notify about being outdated.